src/wrappers/themis/wasm/src/secure_keygen.ts

```diff
@@ -42,17 +42,27 @@ export enum KeyKind {
   EC_PUBLIC = 4,
 }
 
-export class PrivateKey extends Uint8Array {
+export class PrivateKey {
```

Comment:
I had to remove the extends. Officially it is not supported in ES5 to extend buildins. I want to target ES5 instead of ES6 by default to be compatible with the majority of browsers.

Reply:
This led to some minor changes in tests.